curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efada803cbd2bbdab94e332.md

```diff
@@ -40,6 +40,15 @@ assert(
 
 The third image should have a `src` attribute set to `https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg`.
 
+```js
+const catsImg = document.querySelectorAll('figure > img')[1];
+assert(
+  catsImg &&
+    catsImg.getAttribute('src').trim() === 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg'
+);
+```
+The `src` attribute should not have any extra spaces before or after the link. Remove the extra spaces.
+
```

Comment:
````suggestion
```js
const catsImg = document.querySelectorAll('figure > img')[1];
assert(
  catsImg &&
    catsImg.getAttribute('src').trim() === 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg'
);
```

The `src` attribute should not have any extra spaces before or after the link. Remove the extra spaces.

````

the hint text is above the code for the test, I'm not sure you added them in the correct order

Reply:
Yes, the hint must go above the code for the test. I have added this test just after the 'src' attribute value so that the learner first validate that the 'src' is correct and then get further guidance on ensuring there are no extra spaces around it.